hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -592,6 +616,14 @@ private TestInterface targetWithoutFallback() {
     @RequestLine("GET /")
     @Headers("Accept: application/json")
     HystrixCommand<String> command();
+    
+    @RequestLine("GET /")
+    @Headers("Accept: application/json")
+    HystrixCommand<String> enhanced();
+    
+    @RequestLine("GET /{strParam}")
+    @Headers("Accept: application/json")
+    HystrixCommand<String> enhancedWithArg(@Param("strParam") String strParam);
```

Comment:
no need to rename the method when overloading.. just call it `command()`

Having fewer changes draws attention to what's actually changed. For example, the method "command()" refers to the return type which is a hystrix "command" as opposed to a completable.